pages/index.tsx

```diff
@@ -6,20 +6,26 @@ import Link from "next/link";
 import {CandidateTestModel} from "../Models/CandidateTestModel";
 import {TestList} from "../components/CandidateTestView/Tests/TestList";
 
+export let screenWidth = 1080;
 export const testToRender: CandidateTestModel = TestList[1];
+export function reportScreenWidth(): number {
+    screenWidth = screen.width;
+    window.addEventListener('resize', reportScreenWidth);
+    return screenWidth;
+}
+
```

Comment:
If we really need this, then it should probably live on the page where its used. We definitely don't need it on this page, as the text editor isn't here.

Reply:
Was frustrating myself trying to get screen from dom so tried to put it as early on as possible. Removing so no longer an issue :)